lib/core-common/package.json

```diff
@@ -31,8 +31,8 @@
   },
   "files": [
     "dist/**/*",
-    "dll/**/*",
     "types/**/*",
```

Comment:
This actually DOES exist here, I assume the `package.json` was just copied over and over when creating new packages, without looking if it made sense.